gprofiler/log.py

```diff
@@ -75,30 +75,6 @@ def process(self, msg: Any, kwargs: MutableMapping[str, Any]) -> Tuple[Any, Muta
         logging_kwargs["extra"] = extra
         return msg, logging_kwargs
 
-    def debug(self, msg: Any, *args: Any, no_server_log: bool = False, **kwargs: Any) -> None:
```

Comment:
IIRC these were necessary for the `no_server_log: bool` parameter.

Reply:
Decided it's k because orig function accepts `kwargs` and we pop them anyway.

Reply:
The method dimply passes `no_server_log` as it is to the super method, which receives `kwargs` anyway, so this is redundant.
The motivation to this change is mypy errors similar to the one below:
``` bash
gprofiler/log.py:78: error: Signature of "debug" incompatible with supertype "LoggerAdapter"
gprofiler/log.py:78: note:      Superclass:
gprofiler/log.py:78: note:          def debug(self, msg: object, *args: object, exc_info: Union[None, bool, Union[Tuple[Type[BaseException], BaseException, Optional[TracebackType]], Tuple[None, None, None]], BaseException] = ..., stack_info: bool = ..., extra: Optional[Mapping[str, object]] = ..., **kwargs: object) -> None
gprofiler/log.py:78: note:      Subclass:
gprofiler/log.py:78: note:          def debug(self, msg: Any, *args: Any, no_server_log: bool = ..., **kwargs: Any) -> None
```